src/czml3/properties.py

```diff
@@ -41,7 +41,7 @@ class PolylineGlowMaterial(BaseCZMLObject):
 class PolylineArrowMaterial(BaseCZMLObject):
     """"A material that fills the surface of a line with an arrow."""
 
-    color = attr.ib(default=None)
+    polylineArrow = attr.ib(default=None)
```

Comment:
...so the documentation is wrong? Or is there something I'm not following here? https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/PolylineArrowMaterial

Reply:
yes, looks like the manual is wrong

I glanced the correct way of doing it from [this JSON fragment on github](https://github.com/lyqh-ctx/cesiumTx/blob/f6788936274f23c5b1d53671b25c1bcdc78db6e4/examples/czml_polyline.html#L81-L100)

Reply:
xref https://github.com/AnalyticalGraphicsInc/czml-writer/issues/181

Reply:
That example is good enough, thanks a lot!